src/templates/next/next.config.js

```diff
@@ -1,17 +1,25 @@
 module.exports = {
   target: 'serverless',
   webpack(config) {
-    config.module.rules.push({
-      test: /\.svg$/,
-      use: [
-        {
-          loader: '@svgr/webpack',
-          options: {
-            replaceAttrValues: { '#000': 'currentColor' },
+    config.module.rules.push(
+      {
+        test: /\.(png|jpe?g|gif)$/i,
```

Comment:
do we need this?
didn't know that next didn't handle these files automatically

Reply:
NextJS doesn't handle the image import, in order to do it, it needs the nextjs-image plugin to work properly, which basically does the same thing we have here.

Reply:
can I close this issue?